internal/commands/up.go

```diff
@@ -142,7 +170,7 @@ func configureClient(c *UpCommand, dom *rawLibvirt.Domain, image *images.Image,
 
 		// Disable password authentication on SSH.
 		"sudo sed -i '/PasswordAuthentication/s/yes/no/' /etc/ssh/sshd_config",
-		"sudo systemctl reload ssh",
+		"sudo systemctl restart ssh",
```

Comment:
SSH is already running at that point (see `systemdRestartSSH()`), reloading should do.